examples/tutorial/jupyter/execution/pandas_on_dask/requirements.txt

```diff
@@ -0,0 +1,7 @@
+fsspec
+s3fs
+jupyterlab
+ipywidgets
+tqdm
```

Comment:
```suggestion
```
I believe this should be removed since it is related to the progress bar feature.